cmd/start.go

```diff
@@ -60,6 +60,8 @@ Options passed on start override configuration options only on start and are not
 			defer cancel()
 			err = nd.Start(ctx)
 			if err != nil {
+				stopErr := nd.Stop(ctx)
```

Comment:
This is out of the scope of this PR and unlikely to give us anything, so lets remove this